src/machines.scss

```diff
@@ -77,3 +80,7 @@
 .ct-external-docs-link.pf-c-button.pf-m-link.pf-m-inline {
   padding-left: var(--pf-global--spacer--sm);
 }
+
+.ct-monospace {
+    font-family: var(--pf-global--FontFamily--redhat--monospace);
+}
```

Comment:
With a `.ct-`* level, we should probably have this elsewhere in Cockpit, probably under `lib/`* which gets re-imported to Machines.

And, then, eventually, when PatternFly has their own `.pf-` to do this, we switch to that instead.

But, for this PR, I think this is OK and that can be handled in a follow-up.

What do others here think?

Reply:
Agreed -- this should also become a cockpit PR, and then at some point c-m can bump the pkg/lib version and drop its own copy. But let's not block on that indeed.